src/components/MeetOrganizers.jsx

```diff
@@ -0,0 +1,35 @@
+import { ABOUT_HYDPY } from '@/details';
+import Image from 'next/image';
+
+export default function MeetOrganizers() {
+  return (
+    <div className="flex flex-col lg:flex-row justify-center items-center p-6 pt-12 pb-12 border-t-2 border-gray-300">
+      <div className="max-w-3xl lg:mr-20">
+        <h1 className="text-4xl font-bold text-[#001ccf] dark:text-white mb-4">
+          {ABOUT_HYDPY.title}
+        </h1>
+        <p className="text-lg md:text-xl text-gray-600 dark:text-gray-400 mb-8">
```

Comment:
The font size looks good. Can you update for announcement text in hero banner for consistency?
Creation of Paragraph component for centralisation will help here.

Reply:
Updated fontsize in hero